Publish experimental pKa values

MehtapIsik opened the pull request:
I organized experimental pKa results under `SAMPL6/physical_properties/pKa/experimental_data/`.

davidlmobley commented:
@bannanc - can you review?

bannanc commented on physical_properties/pKa/experimental_data/README.md:
```diff
@@ -0,0 +1,14 @@
+- `pKa_experimental_values.csv` - Mean and SEM of pKa values organized in a table. Up to three pKa values were reported based on how many pKas were measured in the experimental interval of 2-12.
+- `calc_pKa_value_statistics.py` - Python script used to calculate mean and SEM of pKa values.
+- `pKa_replicate_experimental_results.csv` - Experimental results of independent replicates of pKa measurements(N=3) and names of experimental reports (PDF files can be found in `experiment_reports\` directory).
+- `experiment_reports\` - This directory contains reports generated by Sirius T3 software for the results of pKa measurements. Refer to `pKa_replicate_experimental_results.csv` for matching SAMPL6 molecule IDs to names of report files.
```
This seems like A LOT of work just to find the correct eperimental report to read. Could you rename the `experiment_reports/` files to include the SAMPL6 molecule ID? 
(as a side note the '\' in this line is the wrong one for directory separation, should be '/'

MehtapIsik replied:
I added SAMPL6 molecule IDs as prefixes to report files.

bannanc replied:
perfect!

bannanc commented on physical_properties/pKa/experimental_data/experimental_ID_and_SAMPL6_molecule_ID_table.csv:
```diff
@@ -0,0 +1 @@
D06,SM21
D07,SM22
D08,SM23
D09,SM24
```
Why is this only one line? I would guess if people are going to look at this it needs to be human readable? Could you separate these by molecule or at least add a comment at the top explaining how this file is organized?

MehtapIsik replied:
It is supposed to be a CSV file of two columns. I fixed this problem.

bannanc commented on physical_properties/pKa/experimental_data/pKa_experimental_values.csv:
```diff
@@ -0,0 +1,25 @@
+Molecule ID,pKa1 mean,pKa1 SEM,pKa2 mean,pKa2 SEM,pKa3 mean,pKa3 SEM,Assay Type,Experimental Molecule ID
+SM01,9.53,0.01,nan,nan,nan,nan,UV-metric pKa,M01
+SM02,5.03,0.01,nan,nan,nan,nan,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",M02
```
Why are there so many nan? does that mean very few molecules actually have replicates? Could you add a comment to the top of the file explaining what is contained here?

MehtapIsik replied:
Some molecules have one experimental pKa, others have two or three macroscopic pKas. I reported NaN for pKa2 and pKa3 for molecules with only one pKa. If all these NaN values are distracting, I can just leave those cells empty.

bannanc replied:
Oh, I thought this was replicate measurements not separate measurements. That makes more sense. I may have miss read something in the README.

bannanc commented on physical_properties/pKa/experimental_data/pKa_replicate_experimental_results.csv:
```diff
@@ -0,0 +1 @@
SM23,5.45,,,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17J-07008,D08,17J-07008_D08_UV-metric psKa_report
SM24,2.61,,,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17J-06007,D09,17J-06007_D09_UV-metric psKa_report
SM24,2.58,,,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17J-06008,D09,17J-06008_D09_UV-metric psKa_report
SM24,2.60,,,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17J-06009,D09,17J-06009_D09_UV-metric psKa_report
```
Same commend as on `experimental_ID_and_SAMPL6_molecule_ID_table.csv` why is this a single line? Could you explain in the file how we're supposed to read this?

MehtapIsik replied:
This file was supposed to be a CSV with two columns. I corrected it.

MehtapIsik commented:
This branch is ready for merge. @davidlmobley 

I corrected the issues @bannanc pointed to in CSV files and renamed the experimental report files with SAMPL6 molecule ID prefixes.

MehtapIsik commented:
Thank you for reviewing. This version is much tidier.

davidlmobley commented:
Do a release, @MehtapIsik or @andrrizzi ?

andrrizzi commented:
I'm on it.
